libsql-server/src/auth/user_auth_strategies/jwt.rs

```diff
@@ -80,6 +88,20 @@ mod jwt_time {
     }
 }
 
+fn validate_any_jwt(
+    jwt_keys: &Vec<jsonwebtoken::DecodingKey>,
+    jwt: &str,
+) -> Result<Authenticated, AuthError> {
+    for (index, jwt_key) in jwt_keys.iter().enumerate() {
+        let result = validate_jwt(&jwt_key, jwt);
+        if result.is_ok() || index == jwt_keys.len() - 1 {
+            return result;
+        }
+    }
+
+    Err(AuthError::Other)
```

Comment:
in case validation fails, we might be interested in individual errors for each jwt
I suggest that we collect all errors an fold them into a single one, something like https://stackoverflow.com/a/61171459

Reply:
When validation fails, we'll return the last error. The only reason we would ever get this generic one is when there are no JWT keys - which really shouldn't happen.

Reply:
Another thought here is do we want to expose the fact that there are multiple keys to clients? That feels like an internal piece of data that really shouldn't be leaked.

Reply:
I believe it's reachable if the `jwt_keys` Vec is empty.